packages/plugins/src/files/gridfs/gridfs-adapter.ts

```diff
@@ -28,7 +30,22 @@ export const GridFSAdapter: IFileAdapter = {
   version: '1.0.0',
 
   ...FileAdapter,
+  async signUrl(fileUrl: string, mediaId: string, expiry?: number) {
```

Comment:
I think this function should take the whole file object instead of just the URL.

so params should be file and expiry

Reply:
@pozylon, it will not work because in media-type, getUrl is called with both the file and any query parameter supported in the file.url, such as version, for example.
and we can determine that parameter when calling it inside webhook

Reply:
@pozylon  it will not work because in media-type, getUrl is called with both the file and any query parameter supported in the file.url, such as version, for example.
That parameter can not be determined when calling it inside the webhook.